components/menu/index.css

```diff
@@ -127,6 +45,16 @@
 	--spectrum-menu-section-header-line-height-cjk: var(--spectrum-cjk-line-height-100);
 	--spectrum-menu-section-header-font-weight: var(--spectrum-bold-font-weight);
 	--spectrum-menu-section-header-color: var(--spectrum-gray-900);
```

Comment:
This `--spectrum-menu-section-header-color`, I think it's supposed to be `--spectrum-neutral-content-color-default` which ends up aliasing to `--spectrum-gray-800`.